Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -145,33 +145,33 @@ Re-enable this permission in the app settings if you permanently disabled it."</
     <string name="open_link_confirmation">Open this link?</string>
     <string name="open_thread_confirmation">Open this thread?</string>
     <string name="open_archived_thread_confirmation">Open this archived thread?</string>
-    <string name="open_link_failed">No applications were found to open this link</string>
-    <string name="open_file_picker_failed">No file picker applications were found</string>
+    <string name="open_link_failed">Install an app that can open this link first.</string>
+    <string name="open_file_picker_failed">Install a file-picker app first.</string>
     <string name="image_pick_delegate_select_file_picker">Select a file picker to open a file…</string>
 
     <string name="thumbnail_load_failed_network">Error</string>
-    <string name="image_preview_failed">Failed to show image</string>
-    <string name="image_preview_failed_oom">Failed to show image (out of memory)</string>
-    <string name="image_failed_big_image">Failed to show image (deep-zoom loading failed)</string>
+    <string name="image_preview_failed">Could not show image</string>
+    <string name="image_preview_failed_oom">Ran out of memory before image could be shown</string>
+    <string name="image_failed_big_image">Could not load deep-zoom when trying to load image</string>
     <string name="image_not_found">Image not found</string>
     <string name="image_spoiler_filename">Spoilered file</string>
     <string name="image_hidden_filename">Hidden file</string>
     <string name="thread_boards_search_hint">Search</string>
     <string name="thread_load_failed_ssl">HTTPS error</string>
     <string name="thread_load_failed_network">Network error</string>
-    <string name="thread_load_failed_parsing">API parse error</string>
+    <string name="thread_load_failed_parsing">API parsing error</string>
     <string name="thread_load_failed_server">Server error</string>
-    <string name="thread_load_failed_not_found">404 not found</string>
+    <string name="thread_load_failed_not_found">What you are looking for doesn\'t exist</string>
```

Comment:
I think the ``404 not found`` is more in the spirit of 4chan, rather than us being grammatically correct.

Reply:
My rationale is if using a client, the bare-bones experience is not sought after.